openedxstats/settings/base.py

```diff
@@ -115,7 +115,7 @@
 #)
 
 # Whitenoise compression config
-STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```

Comment:
have we verified that this works as same as GZip

Reply:
http://whitenoise.evans.io/en/stable/changelog.html#v4-0